### PR TITLE
[ZEPPELIN-2430] The xAxis text in the line and area chart overlaps

### PR DESCRIPTION
**Pull request by soralee**
### What is this PR for?
Like in the #2133 case, the xAxis text of area chart and line chart is overlapping.
For reusing form, I created a html template file for how to show xLabel and, in `rotate` button checked, show up `input` box for set value of rotate degree (default value is -45).

### What type of PR is it?
[Bug Fix | Improvement]

### What is the Jira issue?
* [ZEPPELIN-2430 - The xlabel text in the line and area graph overlaps](https://issues.apache.org/jira/browse/ZEPPELIN-2430)

### How should this be tested?
1. Click `settings` button in the barChart 
2. Check to whether there are `default`, `rotate`, `hide` button or not.
3. Test to click each button and whether it works well or not for each situation.
4. In rotate button checked, check to set value and showing xAixs rotate degree properly.
5. In the areaChart and lineChart, test with above steps.

### Screenshots (if appropriate)
[Before]
image

image

[After]
z_overlap_in_text

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by soralee** [comment]
CI fail is irrelevant this PR.
```
Tests in error: 
  SecurityRestApiTest.testGetUserList:69 » JsonSyntax java.lang.IllegalStateExce...
  SecurityRestApiTest.testTicket:55 » JsonSyntax java.lang.IllegalStateException...
Tests run: 58, Failures: 5, Errors: 2, Skipped: 1
```

File: zeppelin-web/src/app/visualization/builtins/visualization-barchart.js [hunk 1]
```diff
@@ -115,39 +128,16 @@ export default class BarchartVisualization extends Nvd3ChartVisualization {
...
-        </button>
-
-        <button type="button"
-              class="btn btn-default btn-sm"
-              ng-class="{'active' : this.config.isXLabelStatus('hide')}"
-              ng-click="save('hide')" >
-            Hide
-        </button>
-      </div>`,
+      template: `<ng-include src="'app/visualization/builtins/visualization-dispayXAxis.html'">
```
Comment (Leemoonsoo) [inline zeppelin-web/src/app/visualization/builtins/visualization-barchart.js]: [`template` field understand URL](https://github.com/apache/zeppelin/blob/master/zeppelin-web/src/app/visualization/visualization.js#L156). Could you try it without 'ng-include'?

Comment (soralee) [inline zeppelin-web/src/app/visualization/builtins/visualization-barchart.js]: Oh,I didn't know that. I'll update it. 
Thanks!

**Comment by soralee** [comment]
I removed `ng-include` in bar chart and area char. the `ng-include` in line chart couldn't remove now because there are other `html` source. (I think to improve a logic is other problem.)
@moon, could you review again?

**Comment by Leemoonsoo** [comment]
LGTM and merge to master if no further comments.

**Comment by soralee** [comment]
Fixed CI : https://travis-ci.org/soralee/zeppelin/builds/232368042

**Pull request closed without merging**